qa/L0_server_status/server_status_test.py

```diff
@@ -29,80 +29,87 @@
-
-                self.assertGreater(uptime2, uptime1)
-                self.assertEqual(req_id2, req_id1 + 1)
-
+            for pair in [("localhost:8000", "http"), ("localhost:8001", "grpc")]:
+                model_name = "graphdef_int32_int8_int8"
+                extensions = ['classification', 'sequence', 'model_repository', 'schedule_policy',
+                            'model_configuration', 'system_shared_memory', 'cuda_shared_memory',
+                            'binary_tensor_data', 'statistics']
+                if pair[1] == "http":
+                    triton_client = httpclient.InferenceServerClient(url=pair[0], verbose=True)
+                else:
+                    triton_client = grpcclient.InferenceServerClient(url=pair[0], verbose=True)
+
+                self.assertTrue(triton_client.is_server_live())
+                self.assertTrue(triton_client.is_server_ready())
+                server_metadata = triton_client.get_server_metadata()
+                model_metadata = triton_client.get_model_metadata(model_name)
+
+                if pair[1] == "http":
+                    self.assertEqual(os.environ["TRITON_SERVER_VERSION"],
+                                server_metadata['version'])
+                    self.assertEqual("inference:0", server_metadata['name'])
+                    for ext in extensions:
+                        self.assertTrue(ext in server_metadata['extensions'])
+
+                    self.assertEqual(model_name, model_metadata['name'])
+                else:
+                    self.assertEqual(os.environ["TRITON_SERVER_VERSION"],
+                                     server_metadata.version)
+                    self.assertEqual("inference:0", server_metadata.name)
+                    for ext in extensions:
+                        self.assertTrue(ext in server_metadata.extensions)
+
+                    self.assertEqual(model_name, model_metadata.name)
         except InferenceServerException as ex:
             self.assertTrue(False, "unexpected error {}".format(ex))
 
     def test_unknown_model(self):
         try:
-            for pair in [("localhost:8000", ProtocolType.HTTP), ("localhost:8001", ProtocolType.GRPC)]:
-                server_status = _get_server_status(pair[0], pair[1], "foo")
+            for pair in [("localhost:8000", "http"), ("localhost:8001", "grpc")]:
+                model_name = "foo"
+                if pair[1] == "http":
+                    triton_client = httpclient.InferenceServerClient(
+                        url=pair[0], verbose=True)
+                else:
+                    triton_client = grpcclient.InferenceServerClient(
+                        url=pair[0], verbose=True)
+
+                self.assertTrue(triton_client.is_server_live())
+                self.assertTrue(triton_client.is_server_ready())
+                server_metadata = triton_client.get_server_metadata()
+                model_metadata = triton_client.get_model_metadata(model_name)
```

Comment:
You expect this call to throw an exception so the only code after here is the "self.assertTrue(False, "expected unknown model failure")". From below you can move the code that tests server_metadata since you expect that call to be successful.